support/tech-resources/deep-links-self-serve.md

```diff
@@ -5,152 +5,230 @@ description: "A guide to including iOS universal links and Android App Links in
+Email is often read and engaged with on mobile devices. Both iOS (9 and later) and Android (6.0 and later) provide a way for links in your content to take your users directly into an app that you have created, providing your users with a richer interactive experience.
+The links that connect a user directly to your app are referred to as deep links.
 
-This article describes the various ways you can use deep links with email sent through your SparkPost account.
+This article describes how to configure your content and mobile apps to support deep links.
 
+> iOS uses the term Universal Link. The Android term is "App link". We use the term "deep link" in this article to cover the aspects common to both.
+
+When correctly configured, a deep link takes your user directly from the email to your app, without opening the mobile browser. If the app is not installed, the mobile browser will open and display the specified page instead.
+
+## Setup requirements
+
+- Known URL path(s) that you want to deep link to your application.
+    - When using SparkPost Click Tracking, you  choose the Tracking Domain and the custom link sub-path, explained [here](#tracking).
+
+- Deep linking spec files hosted on your website or CDN, in the correct location for the devices to find. The spec file declares the identity of your app, and which URL paths lead to your app - described [here](#specfiles).
+
+- A mobile app that is registered to handle incoming deep link requests - see [example code](#app-examples).
+
+The deep link will operate only when all of these match correctly. When you're ready, check your setup against the [summary of setup steps](#summary).
+
+##  <a name="app"></a> Writing your mobile app
 The following developer documentation explains the deep linking mechanisms in detail.
-- [Apple's Universal Links](https://developer.apple.com/library/content/documentation/General/Conceptual/AppSearch/UniversalLinks.html) 
+- [Apple's Universal Links](https://developer.apple.com/library/content/documentation/General/Conceptual/AppSearch/UniversalLinks.html)
 
 - [Android App Links](https://developer.android.com/training/app-links/index.html)
 
+## <a name="spec-file"></a>Deep linking spec files
+
+The spec files must be published on your domain(s) and accessible via HTTPS. They may be hosted on a Content Delivery Network (CDN) or a regular web server - examples [here]().
```

Comment:
There's no target link for the "here". That needs resolving before we can publish this page, in my opinion.

Based on the @@ comments you left later, I think you want to include some CDN examples in this page. But for now, is it worth linking to the existing CDN articles?

Reply:
I now have a substantial section for CDNs, will update link.
This will *not* replace or duplicate anything in our existing CDN document, rather it will be additive.